src/NavData.tsx

```diff
@@ -100,6 +100,64 @@ const rootNavData: NavMenu = deepFreeze([
           },
         ],
       },
+      {
+        href: '/how-to',
+        title: 'How To',
+        sections: [
+          {
+            href: '/how-to/set-up',
+            title: 'Set Up',
+            sections: [
+              {
+                title: 'Plural CLI',
+                href: '/how-to/set-up/plural-cli',
+              },
+              {
+                title: 'MGMT Cluster',
```

Comment:
These titles need some real work, the terms are not going to be understood by novice users and it'll be the first thing they see when browsing the docs

Reply:
addressed this by changing the nav name and in the docs wrote `Management (MGMT)` so the reference is clear